101-attestation-provider-create/README.md

```diff
@@ -0,0 +1,10 @@
+# Create Attestation Provider
```

Comment:
add a proper readme - what is this? why would I create one? is there documentation that can help me understand? is this GA?

Reply:
This service is currently only for preview and will be GA in Dec timeframe